cryptoki/tests/basic.rs

```diff
@@ -27,7 +28,9 @@ type Result<T> = std::result::Result<T, ErrorWithStacktrace>;
 #[test]
 #[serial]
 fn sign_verify() -> Result<()> {
-    let (pkcs11, slot) = init_pins();
+    let user_pin = "fedcba";
+    let so_pin = "abcdef";
+    let (pkcs11, slot) = init_pins(so_pin, user_pin);
```

Comment:
Maybe we could defined `USER_PIN` and `SO_PIN` as static values in the Rust code and `init_pins` not taking any parameters (since we don't really care what those values are for tests I think).